modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/GroovyScriptUsesModal.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import {Body, Cell, Head, Row, Table, Text} from '@clayui/core';
+import ClayIcon from '@clayui/icon';
+import ClayLink from '@clayui/link';
+import ClayModal, {useModal} from '@clayui/modal';
+import React from 'react';
+
+import {GroovyScriptUseItem} from './ScriptManagementContainer';
+
+import './GroovyScriptUsesModal.scss';
+
+interface GroovyScriptUsesModalProps {
+	groovyScriptUsesItems: GroovyScriptUseItem[];
+	handleOnClose: (value: boolean) => void;
+}
+
+const tableHeaderItems = [
+	{
+		id: 'scriptSource',
+		name: Liferay.Language.get('script-source'),
+	},
+	{
+		id: 'instanceWebId',
+		name: Liferay.Language.get('instance-web-id'),
+	},
+];
+
+export function GroovyScriptUsesModal({
+	groovyScriptUsesItems,
+	handleOnClose,
+}: GroovyScriptUsesModalProps) {
+	const {observer, onClose} = useModal({
+		onClose: () => {
+			handleOnClose(false);
+		},
+	});
+
+	return (
+		<ClayModal center observer={observer} status="warning">
+			<ClayModal.Header>
+				{Liferay.Language.get('setting-cannot-be-deactivated')}
+			</ClayModal.Header>
+
+			<ClayModal.Body className="lfr__script-management-groovy-script-uses-modal-body">
+				<div className="lfr__script-management-groovy-script-uses-modal-description">
+					<Text color="secondary">
+						{Liferay.Language.get(
+							'resolve-all-active-scripting-uses-before-proceeding'
+						)}
+					</Text>
+
+					<Text color="secondary">
+						{Liferay.Language.get(
+							'these-active-scripting-uses-must-be-resolved-click-to-open-a-new-tab'
+						)}
```

Comment:
Hi @danvb112, I noticed that this modal in Figma can show the sentence in a single line:
 
In my tests I can see a subtle difference between Figma and the implemented modal. We have a line break here, which for me takes away some of the elegance of the modal. I had the opportunity to consult Júlia and she approved the suggestion of using the lg size property in this modal. 

![script-modal](https://github.com/liferay-objects/liferay-portal/assets/61854510/a8954b1f-e591-440f-94a0-cc3aa581cbe4)

Reply:
Hey @igor-franca, thanks for noticing this, I've changed the modal size! :smile: